eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -47,6 +50,33 @@ $headers = @{
 
 $query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
 
+function AddLabels([int] $prNumber, [string] $prLabelString)
+{
+  # Adding labels to the pr.
+  if (-not $prLabelString) {
+    Write-Verbose "There are no labels added to the PR."
+    return
+  }
+
+  # Parse the labels from string to array
+  $prLabels = @($prLabelString.Split(",") | % { $_.Trim() } | ? { return $_ })
```

Comment:
This variable name conflicts with the global prLabels variable to avoid any potential issues I would rename it to something like $labelsToAdd or something like that.